src/resources/views/crud/columns/url.blade.php

```diff
@@ -0,0 +1,32 @@
+@php
```

Comment:
I think the same applies here as for https://github.com/Laravel-Backpack/CRUD/pull/4675/files#r981123782 - could probably do the same thing, but load the `text` column, right?

Reply:
for URL, we need to put <a href="url"> also which I think can't done in "text" field. Please suggest

Reply:
We need to put "href" also I mean

Reply:
Resolved URL as discussed. Added a and href in blade file & include "text" blade.